app/src/main/java/org/kiwix/kiwixmobile/main/KiwixReaderFragment.kt

```diff
@@ -160,11 +160,11 @@ class KiwixReaderFragment : CoreReaderFragment() {
       val positions = JSONArray(zimPositions)
       var i = 0
       getCurrentWebView().loadUrl(UpdateUtils.reformatProviderUrl(urls.getString(i)))
-      getCurrentWebView().scrollY = positions.getInt(i)
+      // getCurrentWebView().scrollY = positions.getInt(i)
```

Comment:
If I understand this code it was restoring the scroll position of the webviews on a restore, probably shouldn't be commented out

Reply:
ah, my bad! Legacy from some debugging..